packages/core/src/BaseSwitch/BaseSwitch.tsx

```diff
@@ -171,15 +181,31 @@ export const HvBaseSwitch = forwardRef<HTMLButtonElement, HvBaseSwitchProps>(
         defaultChecked={defaultChecked}
         classes={{
           root: classes.switch,
-          switchBase: classes.switchBase,
+          switchBase: cx(classes.switchBase),
           checked: classes.checked,
-          track: classes.track,
-          thumb: classes.thumb,
+          track: cx(
+            classes.track,
+            color
+              ? css({
+                  backgroundColor: `${getColor(color, "transparent")}!important`,
```

Comment:
please no. use CSS vars and/or `data-color` instead

Reply:
Same as above, we have no clear way of doing this right now.